### PR TITLE
Add support to eval from .py and .txt file

### PR DESCRIPTION
**Pull request by jigarvarma2k20**
Reopened #313

### CODE REVIEW

File: userge/plugins/builtin/executor/__main__.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ async def eval_(message: Message):
         if t.done():
             del _EVAL_TASKS[t]
 
-    flags = message.flags
+    flags = message.flags or message.input_str
```
Comment (Krishna-Singhal) [inline userge/plugins/builtin/executor/__main__.py]: is there any need of `message.input_str` ?

File: userge/plugins/builtin/executor/__main__.py [hunk 2]
```diff
@@ -144,7 +144,19 @@ async def eval_(message: Message):
         await message.edit(f"Canceled eval task [{t_id}] !", del_in=5)
         return
 
-    cmd = message.filtered_input_str
+    cmd = False
+    replied = message.reply_to_message
+    if (replied and replied.document
```
Comment (Krishna-Singhal) [inline userge/plugins/builtin/executor/__main__.py]: you need to change [this](https://github.com/UsergeTeam/Userge/blob/adc1bdc8dc8d5e7021cdb75d0fcf98e700b7a1f4/userge/plugins/builtin/executor/__main__.py#L47)

Comment (jigarvarma2k20) [reply userge/plugins/builtin/executor/__main__.py]: Only there??, No need changes that i made here?

File: userge/plugins/builtin/executor/__main__.py [hunk 3]
```diff
@@ -47,7 +47,18 @@ def getpgid(arg: Any) -> Any:
...
+        cmd = False
+        replied = message.reply_to_message
+        if (replied and replied.document
+                and replied.document.file_name.endswith(
+                    ('.txt', '.py'))
+                and replied.document.file_size <= 2097152):
+            dl_loc = await replied.download()
+            with open(dl_loc, "r") as jv:
+                cmd = jv.read()
+            os.remove(dl_loc)
```
Comment (Krishna-Singhal) [inline userge/plugins/builtin/executor/__main__.py:62]: do not download file here, just do `cmd = True`

Comment (jigarvarma2k20) [reply userge/plugins/builtin/executor/__main__.py:62]: Oohk

Comment (Jigarvarma2005) [reply userge/plugins/builtin/executor/__main__.py]: Done

Comment (jigarvarma2k20) [reply userge/plugins/builtin/executor/__main__.py:62]: > do not download file here, just do `cmd = True`

Done

File: userge/plugins/builtin/executor/__main__.py [hunk 4]
```diff
@@ -53,10 +53,7 @@ async def wrapper(message: Message) -> None:
                 and replied.document.file_name.endswith(
                     ('.txt', '.py'))
                 and replied.document.file_size <= 2097152):
```
Comment (Krishna-Singhal) [inline userge/plugins/builtin/executor/__main__.py:57]: add 

```python
and func.__name__ == "_eval"
```

cuz file support is only for eval command

Comment (Krishna-Singhal) [reply userge/plugins/builtin/executor/__main__.py:57]: also add [these](https://github.com/UsergeTeam/Userge/blob/8f0df28860e362627e113881e06c345d40bf6b98/userge/plugins/builtin/executor/__main__.py#L62-L64) 3 lines [here](https://github.com/UsergeTeam/Userge/blob/8f0df28860e362627e113881e06c345d40bf6b98/userge/plugins/builtin/executor/__main__.py#L168).

Comment (jigarvarma2k20) [reply userge/plugins/builtin/executor/__main__.py:57]: > also add [these](https://github.com/UsergeTeam/Userge/blob/8f0df28860e362627e113881e06c345d40bf6b98/userge/plugins/builtin/executor/__main__.py#L62-L64) 3 lines [here](https://github.com/UsergeTeam/Userge/blob/8f0df28860e362627e113881e06c345d40bf6b98/userge/plugins/builtin/executor/__main__.py#L168).

done

**Review by Krishna-Singhal — APPROVED** [review]

**Comment by rking32** [comment]
thanks @Jigarvarma2005 and @Krishna-Singhal :)

**Pull request merged**